src/frontend/login_page/components/LoginPage.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import { Link } from 'react-router-dom';
+import CSS from 'csstype';
+
+export interface LoginPageState {
+    username: string,
+    password: string,
+    submitted: boolean
+}
+
+interface LoginPageProp {
+
+}
+
+class LoginPage extends React.Component <LoginPageProp, LoginPageState>{
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            username: '',
+            password: '',
+            submitted: false
+        };
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange = (e: any) => {
```

Comment:
According to the top comment for https://stackoverflow.com/questions/40676343/typescript-input-onchange-event-target-value, the type for `e` is `React.ChangeEvent<HTMLInputElement>`. You can try whether it works. Same with the `e` in `handleSubmit` .